chain-signatures/keys/src/hpke.rs

```diff
@@ -37,6 +38,14 @@ pub struct PublicKey(<Kem as hpke::Kem>::PublicKey);
 #[derive(Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub struct SecretKey(<Kem as hpke::Kem>::PrivateKey);
 
+impl fmt::Debug for SecretKey {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("SecretKey")
+            .field("key", &hex::encode(self.to_bytes()))
```

Comment:
This looks dangerous. Should we take a few characters instead of the whole key?

Reply:
We already have a debug for the sign secret key but doesn't hurt to take a few characters off